### PR TITLE
 feat($docs): Sync changes in Chinese & Translate docs-'local development'

### PR DESCRIPTION
**Pull request by Mister-Hope**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [ ] Firefox
- [x] Safari
- [x] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

**Comment by Mister-Hope** [comment]
It is almost the same as #1882 , with two more sync on translate and space problem solved.

File: packages/docs/docs/zh/miscellaneous/local-development.md [hunk 1]
```diff
@@ -2,4 +2,100 @@
...
+## 摘要
+
+如果您看到此处，您可能会对改进 VuePress 核心感兴趣。
+
+VuePress正在使用包含了 [Yarn 工作区](https://yarnpkg.com/zh-Hans/docs/workspaces/) 和 [Lerna](https://github.com/lerna/lerna) 的一个组合。
+
+## 初始化程序包
+
+```bash
+ yarn bootstrap // 它将运行并安装根目录所有的程序包的子文件夹
```
Comment (ulivz) [inline packages/docs/docs/zh/miscellaneous/local-development.md]: 程序包，直接使用 package 会好点，不是所有的词语都需要翻译。

File: packages/docs/docs/zh/miscellaneous/local-development.md [hunk 2]
```diff
@@ -2,4 +2,100 @@
...
+
+它将重组工作空间根目录中的所有依赖项并链接所有程序包。
+
+通过运行以下命令来检查链接：
+
+```bash
+    ls -la node_modules/@vuepress
+```
+
+您将全部链接完成。
```
Comment (ulivz) [inline packages/docs/docs/zh/miscellaneous/local-development.md]: 直接用“你”就好了，技术文档显得太客气会有点奇怪。

**Comment by Mister-Hope** [comment]
FIxed! 😄

**Comment by ulivz** [comment]
After I seriously review the translation, I can basically confirmed that this is modified from the translation from Google.

**Pull request merged**